### DISCUSSION TITLE
Wifi and Bluetooth dont work out of box on intel adapter. helpfull tips included

### DISCUSSION THREAD

**Discussion by urufudev** [Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3#8] [Q&A] [answered]
Hi, i have install but wifi and bluetooth dont work
and u have opencore efi folder and can share pls?

**Comment by Leo3213** [comment]
On this laptop i had to use clover because i couldnt get the battery patched with open core.

but here is my efi anyway :)

Also are they the intel ones that came with it?
[EFI.zip](https://github.com/Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3/files/6418735/EFI.zip)

**Comment by Leo3213** [comment]
On this laptop i had to use clover because i couldnt get the battery patched with open core.

but here is my efi anyway :)

Also are they the intel ones that came with it?
[EFI.zip](https://github.com/Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3/files/6418735/EFI.zip)

Interfaces:
en1:
  Card Type:	AirPort Extreme  (0x14E4, 0x117)
  Firmware Version:	Broadcom BCM43xx 1.0 (7.77.111.1 AirPortDriverBrcmNIC-1680.7)
  MAC Address:	48:d7:05:b6:fa:67
  Locale:	FCC
  Country Code:	US
  Supported PHY Modes:	802.11 a/b/g/n/ac
  Supported Channels:	1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 36, 40, 44, 48, 52, 56, 60, 64, 100, 104, 108, 112, 116, 120, 124, 128, 132, 136, 140, 144, 149, 153, 157, 161, 165
  Wake On Wireless:	Supported
  AirDrop:	Supported
  AirDrop Channel:	149
  Auto Unlock:	Supported
  Status:	Connected

**Comment by Leo3213** [comment]
On this laptop i had to use clover because i couldnt get the battery patched with open core.

but here is my efi anyway :)

Also are they the intel ones that came with it?
[EFI.zip](https://github.com/Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3/files/6418735/EFI.zip)

Interfaces:
en1:
  Card Type:	AirPort Extreme  (0x14E4, 0x117)
  Firmware Version:	Broadcom BCM43xx 1.0 (7.77.111.1 AirPortDriverBrcmNIC-1680.7)
  MAC Address:	48:d7:05:b6:fa:67
  Locale:	FCC
  Country Code:	US
  Supported PHY Modes:	802.11 a/b/g/n/ac
  Supported Channels:	1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 36, 40, 44, 48, 52, 56, 60, 64, 100, 104, 108, 112, 116, 120, 124, 128, 132, 136, 140, 144, 149, 153, 157, 161, 165
  Wake On Wireless:	Supported
  AirDrop:	Supported
  AirDrop Channel:	149
  Auto Unlock:	Supported
  Status:	Connected

**Comment by Leo3213** [comment]
Sorry i was a little late im in collage lol did you fix your problem?

**Comment by urufudev** [comment]
> Sorry i was a little late im in collage lol did you fix your problem?

I continue with the problem, I had to buy a compatible plug and play wifi card

and one more question, did you manage to map all the usb ports?

**Comment by Leo3213** [comment]
I did not.

Im really not good at that lol.

which card did you get?

**Comment by Leo3213** [comment]
yeah the card that comes with it will not work its intel lol.

**Comment by urufudev** [comment]
I buy this

Plug & Play Hackintosh M.2 NGFF WLAN BCM94360NG 802.11ac Bluetooth 4.0 Tarjeta WiFi para PC Catalina Mac OS Native Support macOS AirDrop Continuity Handoff Better BCM94352Z DW1560 Soporte Intel NUC https://www.amazon.com/dp/B083YXS7VF/ref=cm_sw_r_cp_api_glt_fabc_C60YTZJCNCPM5KPAPHY1

I hope it works for me

**Comment by Leo3213** [comment, marked as answer]
Hey what i bought was these legit apple adapters

one it the wifi blutooth card and the other converts apples propriatery conecter to the m.2 wifi conector on the laptop :)

apple watch unlock even works

https://www.ebay.com/itm/332861468110

and 

https://www.ebay.com/itm/293023708244

note though with the adapter it barely fits with the cover but it does fit with the cover on

**Comment by Leo3213** [comment]
ive always just bought the apple once becase

A its from an actual mac so it will work lol
b its out of the box handoff/continuity support :)

**Comment by Leo3213** [comment]
note tho on this adapter if you use it on windows you will need to install the bluetooth driver from apple bootcamp windows drivers

below is the bluetooth driver i installed on my laptop for bluetooth to work on windows

[brodcom apple adaper windows drivers.zip](https://github.com/Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3/files/6448465/brodcom.apple.adaper.windows.drivers.zip)

i also included a wifi driver that came with it just in case :)

**Comment by Leo3213** [comment]
If you have anymore questions use discussions :)

**Comment by robi62** [comment]
the intel wifi card works with the correct kext

**Reply by Leo3213** [reply]
Oh thx for the update
